dts/arm/st/f1/stm32f105Xc.dtsi

```diff
@@ -18,5 +18,24 @@
 				reg = <0x08000000 DT_SIZE_K(256)>;
 			};
 		};
+
+		usbotg_fs: usb@50000000 {
+			compatible = "st,stm32-otgfs";
+			reg = <0x50000000 0x40000>;
+			interrupts = <67 0>;
+			interrupt-names = "otgfs";
+			num-bidir-endpoints = <4>;
+			ram-size = <1280>;
+			clocks = <&rcc STM32_CLOCK_BUS_AHB1 0x00001000>;
+			phys = <&otgfs_phy>;
+			status = "disabled";
+			label = "OTGFS";
+		};
+	};
+
+	otgfs_phy: otgfs_phy {
+		compatible = "usb-nop-xceiv";
+		#phy-cells = <0>;
+		label = "OTGFS_PHY";
 	};
```

Comment:
This should be moved to stm32f105.dtsi so it could be available to stm32f107.dtsi